libpod/image/filters.go

```diff
@@ -29,6 +29,26 @@ func CreatedBeforeFilter(createTime time.Time) ResultFilter {
 	}
 }
 
+// IntermediateFilter returns filter for intermediate images (i.e., images
+// without children).
+func (ir *Runtime) IntermediateFilter(ctx context.Context, images []*Image) (ResultFilter, error) {
```

Comment:
This is not new, but it’s confusing how `Runtime.IntermediateFilter` and `layerTree.isIntermediate` have different semantics. It would be nice for the two to use different names — not that I know _what_ names.